KeyboardLayoutGuide/KeyboardLayoutGuide/Keyboard+LayoutGuide.swift

```diff
@@ -51,11 +56,21 @@ open class KeyboardLayoutGuide: UILayoutGuide {
         guard let view = owningView else {
             return
         }
-        NSLayoutConstraint.activate([
-            heightAnchor.constraint(equalToConstant: 0),
-            leftAnchor.constraint(equalTo: view.leftAnchor),
-            rightAnchor.constraint(equalTo: view.rightAnchor),
-            ])
+        
+        if #available(iOS 11.0, *) {
+            NSLayoutConstraint.activate([
+                heightAnchor.constraint(equalToConstant: Keyboard.shared.currentHeight),
```

Comment:
Better to conditionally calculate only the constant in if-else here, leaving a single `NSLayoutConstraint.activate([...])` part (no repeating code)